src/interface/src/app/plan/create-scenarios/constraints-panel/constraints-panel.component.ts

```diff
@@ -41,7 +41,7 @@ export class ConstraintsPanelComponent {
           // Maximum slope allowed for planning area
           maxSlope: [, [Validators.min(0), Validators.max(100)]],
           // Minimum distance from road allowed for planning area
-          minDistanceFromRoad: [, [Validators.min(0)]],
+          minDistanceFromRoad: [, [Validators.min(0), Validators.max(100000)]],
```

Comment:
I suppose this is a question for other people, but I wonder if it's worth displaying the max in the label.